File: app/src/main/java/org/odddev/fantlab/auth/reg/IRegView.java
package org.odddev.fantlab.auth.reg;

import com.arellomobile.mvp.MvpView;

/**
 * @author kenrube
 * @since 18.09.16
 */

interface IRegView extends MvpView {

    void showRegResult(boolean registered);

    void showError(String error);
}

File: app/src/main/java/org/odddev/fantlab/auth/reg/IRegView.kt
package org.odddev.fantlab.auth.reg

import com.arellomobile.mvp.MvpView

/**
 * @author kenrube
 * *
 * @since 18.09.16
 */

interface IRegView : MvpView {

	fun showRegResult(registered: Boolean)

	fun showError(error: String)
}
